e2e/api_test.go

```diff
@@ -175,6 +189,14 @@ func TestE2E_StatusEndpoints(t *testing.T) {
 
 			checkEvents(t, taskStatuses, fakeFailureTaskName, tc.expectFailureTask)
 			checkEvents(t, taskStatuses, fakeSuccessTaskName, tc.expectSuccessTask)
+
+			task, ok := taskStatuses[disabledTaskName]
+			if tc.expectDisabledTask {
+				require.True(t, ok)
+				assert.Nil(t, task.Events)
+			} else {
+				require.False(t, ok)
```

Comment:
Both of the `require` reads to me that they should be `assert` because it's a part of the testing expected values instead of test setup.

Reply:
Yes, agree! Thank you for catching this :)